legal-api/src/legal_api/resources/v2/admin/configuration.py

```diff
@@ -28,8 +28,19 @@
 @cross_origin(origin='*')
 @jwt.has_one_of_roles([UserRoles.staff])
 def get_configurations():
-    """Return a list of configurations."""
-    configurations = Configuration.all()
+    """Return a list of configurations, optionally filtered by names."""
+    filter_names = request.args.get('names', None)
+    if filter_names:
+        names_list = [name.strip() for name in filter_names.split(',') if name.strip()]
+        if not names_list:
+            return {'message': 'Configuration names are invalid'}, HTTPStatus.BAD_REQUEST
+
+        configurations = Configuration.find_by_names(names_list)
+        if not configurations:
```

Comment:
will that be a good idea to add a local names validation before sending a query? something like this
```Python
invalid_names = [name for name in name_list if not instance(name, Configuration.Names)]
if not invalid_names:
    # send queries and return results
else:
    # return a message including the invalid names, or maybe just say some of the names are invalid
```